src/editor/profiler_ui.cpp

```diff
@@ -378,8 +378,8 @@ struct ProfilerUIImpl : public ProfilerUI
 		Lumix::Array<float> m_frames;
 		struct Hit
 		{
-			Lumix::uint64 start;
```

Comment:
> Was is something about unsigned long to unsigned long long conversion?

Yeah, something like that. bgfx has [interface](https://github.com/nem0/LumixEngine/blob/master/external/bgfx/include/bgfx/bgfx.h#L315) where some methods take `uint64_t` as parameter. Lumix's code cannot override it with it's own `uint64`.